### DISCUSSION TITLE
I don't get any guide at all

### DISCUSSION THREAD

**Discussion by Stuggy** [iptv-org/community#498] [Q&A] [answered]
New install on windows:

```
npm run grab -- --site=tvguide.com

> <grab>
> ts-node scripts/commands/epg/grab.ts --site=tvguide.com

> staring...
config:
  output: guide.xml
  maxConnections: 1
  gzip: false
  site: tvguide.com
loading channels...
  found 0 channels
creating queue...
  added 0 items
run #1:
  done in 00h 00m 00s
finished`
```

I can't work out what is going on.  I tried a few different sites and always get 0 results.

**Comment by SecretHero2019** [comment]
Same here

**Comment by PopeyeTheSai10r** [comment]
I think the issue is related to the channels not being found.

```
loading channels...
  found 0 channels
```

Using the following format is currently working for me on a Windows installation.
`npm run grab -- --channels=sites/example.com/example.com.channels.xml`



https://github.com/iptv-org/epg#usage says
>   -c, --channels <path>         Path to *.channels.xml file (required if the "--site" attribute is
>                                 not specified)

**Reply by SecretHero2019** [reply]
Thanks.

**Comment by Stuggy** [comment, marked as answer]
I got it to work.  If I have the --site option filled in it doesn't seem to work with or without supplying the channels.xml full path including file name.

If I omit the --site command and provided the full --channels along with path and file name it works.

**Reply by PopeyeTheSai10r** [reply]
I am glad to hear the workaround got you the results.
This is currently being tracked as a bug by https://github.com/iptv-org/epg/issues/2182.
